docs/model_library/water_treatment/index.rst

```diff
@@ -0,0 +1,780 @@
+      - :math:`{\frac{kW}{m^2K}}`
+      - None
+  
+    * - :math:`{U_{ph}}`
+      - Overall heat transfer coefficient of the preheater
+      - :math:`{\frac{kW}{m^2K}}`
+      - None
+
+    * - :math:`{A_{evap}}`
+      - Area of the evaporator
+      - :math:`{m^2}`
+      - i
+
+    * - :math:`{A_{ph}}`
+      - Area of the preheater
+      - :math:`{m^2}`
+      - None
+
+    * - :math:`{C_{p}^{feed}}`
+      - Specific heat capacity of the feed water
+      - :math:`{kJ/(kg ^{\circ}C)}`
+      - None
+
+    * - :math:`{C_{p}^{vapor}}`
+      - Specific heat capacity of vapor
+      - :math:`{kJ/(kg ^{\circ}C)}`
+      - None
+
+    * - :math:`{C_{p}^{mix}}`
+      - Specific heat capacity of water from the outlet of the mixer
+      - :math:`{kJ/(kg ^{\circ}C)}`
+      - None
+
+    * - :math:`{{\Delta}T }`
+      - Approach temperatures
+      - :math:`{^{\circ}C}`
+      - None
+
+    * - :math:`{{\Delta}P }`
+      - Minimum pressure difference between evaporator stages
+      - :math:`{kPa}`
+      - None
+
+    * - :math:`{C_{elec}}`
+      - Cost of electricity  
+      - :math:`{kUSD/kW year}`
+      - None
+
+    * - :math:`{CAPEX_{evap}}`
+      - Capital cost for all evaporators
+      - :math:`{kUSD}`
+      - None
+
+    * - :math:`{CAPEX_{ph}}`
+      - Capital cost for Preheater
+      - :math:`{kUSD}`
+      - None
+
+    * - :math:`{CAPEX_{compr}}`
+      - Capital cost for compressor 
+      - :math:`{kUSD}`
+      - None
+
+    * - :math:`{OPEX_{ann}}`
+      - Annualized operating cost
+      - :math:`{kUSD/year}`
+      - None
+
+    * - :math:`{CAPEX_{ann}}`
+      - Annualized capital cost
+      - :math:`{kUSD/year}`
+      - None
+
+
+
+
+
+Modeling Equations
+-------------------
+
+Evaporator model
+++++++++++++++++
+
+Flow balance in the evaporators:
+
+.. math::
+
+    F_{in} = F_{brine}^{(I)} + F_{vapor}^{(I)}
+
+    F_{in}^{(i+1)} = F_{brine}^{(i)} + F_{vapor}^{(i)} \quad \forall i < I
+
+Flow matching between super heated vapor and flow of vapor from the evaporator:
+
+.. math:: 
+
+    F_{spv} = F_{vapor}^{(I)}
+
+Mass balance in the evaporators:
+
+.. math:: 
+
+    F_{in}Q_{in} = F_{brine}^{(I)}Q_{brine}^{(I)}
+
+    F_{brine}^{(i+1)}Q_{brine}^{(i+1)}=F_{brine}^{(i)}Q_{brine}^{(i)} \quad \forall i < I
+
+Calculating mass fraction of salt from salt salinity:
+
+.. math:: 
+
+    X_{s}^{in} = 0.001 Q_{in}
+
+    X_s^{(i)} = 0.001 Q_{brine}^{(i)} \quad \forall i \in \{0,...,I\}
+
+Energy balance in the evaporator:
+
+.. math:: 
+
+    E^{(I)} + F_{in}H_{in} = F_{brine}^{(I)}H_{brine}^{(I)} + F_{vapor}^{(I)}H_{vapor}^{(I)}
+
+    E^{(i)} + F_{brine}^{(i+1)}H_{brine}^{(i+1)} = F_{brine}^{(i)}H_{brine}^{(i)} + F_{vapor}^{(i)}H_{vapor}^{(i)} \quad \forall i < I
+
+    E^{(1)} =  F_{spv}C_p^{vapor}(T_{spv} - T_{cond}^{(1)}) + F_{spv}(H_{cond}^{vap(1)} - H_{cond}^{(1)})
+
+    E^{(i)} = F_{vapor}^{(i-1)} \lambda^{(i)} \quad \forall i \in \{2,...,I\}
+
+Thermodynamic Relations
```

Comment:
```suggestion
**Thermodynamic Relations**
```
To distinguish it from the surrounding content, I think that "Thermodynamic Relations" should be written in bold text.